service/src/main/java/uk/gov/hmcts/reform/fpl/model/Others.java

```diff
@@ -25,13 +26,13 @@ public static Others from(List<Element<Other>> allOthers) {
 
         final LinkedList<Element<Other>> others = new LinkedList<>(Optional.ofNullable(allOthers).orElse(emptyList()));
         return Others.builder()
-            .firstOther(ofNullable(others.pollFirst()).map(Element::getValue).orElse(null))
+            .firstOther(ofNullable(others.pollFirst()).map(Element::getValue).filter(not(Other::isEmpty)).orElse(null))
```

Comment:
thoughts on something like this?
```java
        Element<Other> first;
        do {
            first = others.pollFirst();
        } while (null == first && !others.isEmpty());

        return Others.builder()
            .firstOther(ofNullable(first).map(Element::getValue).orElse(null))
            .additionalOthers(others)
            .build();
```
it empties the list until it finds the first non null other or reaches the end of the list